eos/effects/industrialcoreeffect2.py

```diff
@@ -1,29 +1,100 @@
 # industrialCoreEffect2
 #
 # Used by:
-# Variations of module: Industrial Core I (2 of 2)
+# Module: Industrial Core I
```

Comment:
I didn't change it. :)

I'll often set them to *SOMETHING* but that's only for my own reference.